LuaParser/src/Parse/LuaParser.cpp

```diff
@@ -399,6 +413,8 @@ void LuaParser::LabelStatement() {
 
     CheckAndNext(TK_DBCOLON);
 
+    TestAndNext(';');
```

Comment:
label statements should never end with a semicolon.

Reply:
Removed, and added the exception to not add one to the analyzer